.circleci/config.yml

```diff
@@ -46,63 +20,27 @@ jobs:
       - install_sccache
       - restore_sccache_cache
       - run:
-          name: "Beta Build"
+          name: "Beta Build & Test"
           command: |
-            rustup run ${DEFAULT_TOOLCHAIN} cargo build
+            make ci
       - save_sccache_cache
-      - store_artifacts:
-          path: ./target/
   beta-test:
     working_directory: ~/workspace
     machine:
       image: ubuntu-1604:201903-01
     environment:
       - RUST_TEST_THREADS: "8"
       - DEFAULT_TOOLCHAIN: "beta"
```

Comment:
Change this to `RUST_TOOLCHAIN`?

We use the word "DEFAULT" here, which is a weird qualifier if we never do anything different :thinking: